README.md

```diff
@@ -257,14 +263,22 @@ Binaries (extract to C:\Projects\tob_libraries)
 * [LLVM 5](https://s3.amazonaws.com/cxx-common/libraries-llvm50-windows10-amd64.7z)
```

Comment:
forgot to update the deps part

Reply:
No, this is still correct. We haven't rebuilt cxx-common for Windows yet.

Reply:
I won't have access to the Windows machine I uses for this until I get back home in a week. Will close for now